careerfairsystems/arkad-app-2018
# Fix Studie C Floor 2 map

## Change description

Adding the new map of Studiecentrum Floor 2. The previous map was missing booth number 191.

## How to verify

Go to the maps screen and navigate to Studiecentrum Floor 2
- Verify that booth number 191 is visible on the map
- Verify that pressing Lunds Tekniska Högskola or LTH Studie- och karriärvägledare shows them on the map
- Verify that the button to show on map from company details screen work with the above companies
- Verify that all other booth positions on the map work by clicking some random companies in the list

## Issues fixed

This fixes #117.
